core/oidc.go

```diff
@@ -453,6 +455,10 @@ func (o *OIDC) token(ctx context.Context, req *tokenRequest, handler func(req *T
 		return nil, &httpError{Code: http.StatusInternalServerError, Message: "internal error", CauseMsg: "session authorization is nil"}
 	}
 
+	var nonce string
+	if sess.Request != nil {
+		nonce = sess.Request.Nonce
+	}
```

Comment:
`PrefillIDToken` was not previously doing this `nil` check. Is there a case where `sess.Request == nil` will be true now?

Reply:
I mainly did this because it caused test failures, but i can fix the test